FEAT: Convert status cli as a subcommand of crio (#6557)
cri-o/cri-o: docs/crio.8.md

**--force, -f**: force wipe by skipping the version check

## status

> display status information

Reviewer: ```suggestion
Display status information
```

Author: Done.

**--socket, -s**="": absolute path to the unix socket (default: /var/run/crio/crio.sock)

### config, c

> Show the configuration of CRI-O as TOML string.

Reviewer: ```suggestion
Show the configuration of CRI-O as a TOML string.
```

Author: Done.

### containers, container, cs, s

Display detailed information about the provided container ID.

**--id, -i**="": the container ID

### info, i

> Retrieve generic information about CRI-O, like the cgroup and storage driver.

Reviewer: Soft suggestion
```suggestion
Retrieve generic information about CRI-O, such as the cgroup and storage driver.
```

Author: Done.
